app/models/service_provider_request.rb

```diff
@@ -5,7 +5,7 @@ class ServiceProviderRequest
   # since these objects are serialized to/from Redis and may be present
   # upon deployment
```

Comment:
@KeithNava  I am not sure what precautions are needed. I would ask to ensure you handled everything for a smooth deployment since you made some modifications.